### ISSUE TITLE
New Exchange: AEX

### ISSUE THREAD

**Issue by jjhesk** [ccxt/ccxt#1564]
api doc: https://www.aex.com/page/api_detailed.html
url: https://www.aex.com
fee: --
volume: --
address: Vistra Corporate Services Centre, Suite 33, 2nd Floor, Eden Plaza, Eden Island, Under British Administration
company info: Bit World Investments Limited, https://coinrate.net/terms
Registration: --
based country: British
whitepaper: --
priority: low
github group: --
note: ranked top : NA
type: centrailzed ex

**Comment by jjhesk** [comment]
Quotes API are not authorized to use, and transaction class API require public and private keys.

Because the server has anti-CC attack strategy, every 60 seconds the number of calls can not be more than 120 times, more than part will be blocked by the firewall.

Since our distributed CDN limits non-browser requests, you may need to manually set USER-AGENT to avoid blocking the firewall when calling APIs.

**Comment by kroitor** [comment]
@catalsdevelop hi!

> when will this exchange merge into master? It's a very famous exchange in China.

As soon as someone submits a pull request with it. Or we will do it sooner than that.
Basically, your question is answered here: https://github.com/ccxt/ccxt/wiki/FAQ#when-will-you-add-feature-foo-for-exchange-bar--whats-the-estimated-time-when-should-we-expect-this

Hope to do it sooner.

BTW, your contributions are welcome ;)

https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
